tests/Unit/Logs/TestHook/DeprecationTest.php

```diff
@@ -33,6 +33,7 @@ protected function createEvent(): DeprecationTriggered
             123,
             false,
             false,
+            false,
```

Comment:
Why did you add this?

Reply:
I have testet i locally by adding these line to composer.json `,
    "minimum-stability": "dev",
    "prefer-stable": false`

With that i got more dev-packages that i wanted.
And there is a new parameter for that function call in stability `dev` that broke testing locally.

Reply:
I did revert the change. But it will be needed in the future i guess.